tests/samples/grundsteuer_sample_data.py

```diff
@@ -2,6 +2,74 @@
 import datetime
 
 from erica.request_processing.erica_input.v2.grundsteuer_input import GrundsteuerData
+from erica.request_processing.erica_input.v2.grundsteuer_input_gebaeude import Gebaeude
+
+
+class SampleGebaeude:
```

Comment:
I really really like it. Could by any chance also to this for eigentuemer? 🙈

Reply:
Remind me to revisit this later. If there's still time once I'm done with grundstueck, gladly.